test/integration/cart_integration_test.rb

```diff
@@ -14,10 +19,10 @@ class CartIntegrationTest < ActionDispatch::IntegrationTest
   end
 
   test 'an unauthorized user can add an item to the cart' do
-    item = Item.create(title: 'coffee', description: 'black nectar of the gods', price: 1200)
+    item = create(:item)
     visit "/items/#{item.id}"
 
-    click_link_or_button('Add to Cart')
+    click_button 'Add to Cart'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.